app/components/recirculation-prefooter.js

```diff
@@ -24,6 +24,39 @@ export default Component.extend(
 		isVisible: false,
 		liftigniter: inject.service(),
 		i18n: inject.service(),
+		hasNoLiftigniterSponsoredItem: computed('items', function () {
+			return !this.get('items').some((item) => item.presented_by);
+		}),
+		isInRightCountry: false,
+		shouldShowPlista: computed('hasNoLiftigniterSponsoredItem', function () {
+			return ['AU', 'NZ'].indexOf(M.geo.country) > -1 && this.get('hasNoLiftigniterSponsoredItem');
+		}),
+		fetchPlista() {
+			const plistaURL = 'http://farm.plista.com/recommendation/?publickey=845c651d11cf72a0f766713f&widgetname=api' +
+							'&count=1&adcount=1&image[width]=583&image[height]=328';
+			return fetch(plistaURL)
+				.then((response) => response.json())
+				.then((data) => {
+					if (data.length) {
+						return data[0];
+					} else {
+						throw new Error('We haven\'t got PLISTA!');
+					}
+				})
+				.catch((error) => {
+					// If Plista did not return anything, just don't add it to to items
+				});
+		},
+		mapPlista(item) {
+			return {
+				meta: 'wikia-impactfooter',
+				thumbnail: item.img,
```

Comment:
When `fetchPlista()` doesn't return any item (in other words - returns `undefined`), this will throw an error.